profile-widget/diveeventitem.cpp

```diff
@@ -143,21 +158,30 @@ void DiveEventItem::setupPixmap()
 #undef EVENT_PIXMAP_BIGGER
 }
 
-void DiveEventItem::setupToolTipString()
+void DiveEventItem::setupToolTipString(struct gasmix *lastgasmix)
 {
 	// we display the event on screen - so translate
 	QString name = gettextFromC::instance()->tr(internalEvent->name);
 	int value = internalEvent->value;
 	int type = internalEvent->type;
 
 	if (event_is_gaschange(internalEvent)) {
+		struct icd_data icd_data;
 		struct gasmix *mix = get_gasmix_from_event(&displayed_dive, internalEvent);
 		name += ": ";
 		name += gasname(mix);
 
 		/* Do we have an explicit cylinder index?  Show it. */
 		if (internalEvent->gas.index >= 0)
 			name += tr(" (cyl. %1)").arg(internalEvent->gas.index + 1);
+		bool icd = isobaric_counterdiffusion(lastgasmix, mix, &icd_data);
+		if (icd_data.dHe < 0) {
+			if (icd)
+				name += tr("\nICD:ΔHe=%1% ΔN₂=%2%>%3%").arg(icd_data.dHe / 10).arg(icd_data.dN2 / 10).arg(-icd_data.dHe / 50);
```

Comment:
Thanks for the ":" but may I ask for a space after the colon as well?
That's not only my personal preference but simply comes from looking at the other info box items we have today.
It's always:
**Whatever: Data**
(with small number of exceptions for completely different items)

![grafik](https://user-images.githubusercontent.com/25316050/35193631-ae389f00-fea5-11e7-8a63-cf57e43380b9.png)

Reply:
Wow, that was quick... thanks!
I'll now also do a quick test but I expect that this is all ok now.